components/toast/index.css

```diff
@@ -86,12 +184,15 @@ governing permissions and limitations under the License.
   align-self: center;
 
   .spectrum-Button {
+    /* https://caniuse.com/mdn-css_properties_float_flow_relative_values */
     float: inline-end;
```

Comment:
This only works on Firefox.  Looking for a better supported approach.

Reply:
`float: inline-end` and `:dir(ltr) { float: right; }` are the only approaches I found, but both are Firefox only. Should I have `float: right` as a fallback even though it won't work for RTL direction @pfulton?

Reply:
Can you use a browser-non specific version?

Reply:
Hey Bernhard, both options that I have found are only ones I found that are based on directionality. `float: right;` is used currently. I haven't found another approach.

Reply:
Awesome! thanks for checking! `float: right` it is!